src/main/java/duke/command/DeletePatientTaskCommand.java

```diff
@@ -31,19 +31,20 @@
      * @param deleteInfo .
      * @throws DukeException .
      */
-    public DeletePatientTaskCommand(String[] deleteInfo) throws DukeException {
+    public DeletePatientTaskCommand(String deleteInfo) throws DukeException {
 
-        char firstChar = deleteInfo[0].charAt(0);
+        char firstChar = deleteInfo.charAt(0);
         try {
             if (firstChar == '#') {
-                this.patientId = Integer.parseInt(deleteInfo[0].replace("#","").trim());
-                this.taskId = Integer.parseInt(deleteInfo[1]);
+                this.patientId = Integer.parseInt(deleteInfo.replace("#","").trim());
```

Comment:
It might be easier to do `Integer.parseInt(deleteInfo.substring(1))` just to minimize the amount of same code to the same effect.